src/render.js

```diff
@@ -0,0 +1,10 @@
+export const renderPosition = {
```

Comment:
```suggestion
export const RenderPosition = {
```
это енам (перечисление). Оно должно начинаться с большой буквы